components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -89,14 +89,18 @@ const ContributorListTableRow = ({
             />
           )}
           <div className="w-[68%]">
-            <DevProfile
-              company={user?.company || getLastContributedRepo(data)}
-              username={login}
-              hasBorder={!contributor.author_login}
-            />
+            <DevProfile username={login} hasBorder={!contributor.author_login} />
           </div>
           <div className="w-[34%] text-normal text-light-slate-11  h-full">
-            <div className="flex gap-x-3">{<p>{getLastContributionDate(mergedPrs)}</p>}</div>
+            <div className="flex flex-col gap-x-3">
+              <p>{getLastContributionDate(mergedPrs)}</p>{" "}
+              <p
+                className="text-sm font-normal truncate whitespace-nowrap overflow-hidden overflow-ellipsis text-light-slate-9 md:hidden lg:max-w-[8.12rem]"
```

Comment:
truncate in tailwind is a combination of 
- overflow-hidden
- text-overflow-ellipsis
- whitespace-nowrap

all you need to achieve the truncate here is the `truncate` class

```suggestion
                className="text-sm font-normal truncate text-light-slate-9 md:hidden lg:max-w-[8.12rem]"
```